Graphs/LineGraph/index.js

```diff
@@ -308,7 +306,7 @@ class LineGraph extends XYGraph {
         }
 
         if(yTicks){
-            yAxis.ticks(yTicks);
+            //yAxis.ticks(yTicks);
```

Comment:
let's revert this

Reply:
Sorry Natalia, by mistake I have uploaded testing code here, I am reverting the test code.